package.json

```diff
@@ -11,8 +11,6 @@
   },
   "license": "Apache-2.0",
   "author": "ActiveViam",
-  "main": "dist/index.js",
```

Comment:
How does this work? Are you leveraging the fact that `main=dist/index.js` is a default? If that's the case, I would rather keep it explicitly written in our `package.json`.

Reply:
Quoting the ticket description:
> From another perspective [authenticating on the private registry] might be unnecessary if the built version was publicly available without authentication, seeing as the tool is open source.

I thought this was a good idea, and taking it further I think nobody is using `activeui-migration` as a library, so shipping a bundled CLI is enough.
This is why I am moving all dependencies to dev dependencies, and why I am removing `index.ts` and `main`.
Only the `bin` is exposed.

I'll add some testing instructions in the PR description.